src/main/resources/stsjorbsmodResources/localization/spa/JorbsMod-Power-Strings.json

```diff
@@ -64,8 +64,8 @@
   "stsjorbsmod:DamnationPower": {
     "NAME": "Condenación",
     "DESCRIPTIONS": [
-      "Permanentemente #yTransforma la próxima carta jugada este combate.",
-      "Permanentemente #yTransforma la próxima carta jugada este combate, luego #yMejora la carta."
+      "Permanentemente #yTransforma la próxima carta no- [REMOVE_SPACE]#yLegendario jugada este combate.",
+      "Permanentemente #yTransforma la próxima carta no- [REMOVE_SPACE]#yLegendario jugada este combate, después la #yMejora."
```

Comment:
by adding `no-legendaria` as a valid keyword for `Legendary`, this can probably also change.